src/usecases/entertainment/entertainment.ts

```diff
@@ -0,0 +1,51 @@
+import RedditMemes from '../../connectors/reddit/redditMemes';
+import ChuckNorris from '../../connectors/chucknorris/chucknorris';
+import Spotify from '../../connectors/spotify/spotify';
+import UseCase from '../../interfaces/useCase';
+import UseCaseResponse from '../../classes/UseCaseResponse';
+import ProcessedTelegramMessage from '../../classes/ProcessedTelegramMessage';
+import TextResponse from '../../classes/TextResponse';
+import ImageResponse from '../../classes/ImageResponse';
+import EndUseCaseResponse from '../../classes/EndUseCaseResponse';
+
+export default class Entertainment implements UseCase {
```

Comment:
Kannst du die Klasse in `EntertainmentUsecase` umbenennen? (https://github.com/JaydenUlysses/Olaf-the-coolest-of-Bots/tree/dev#folder-structure)